medusa/common.py

```diff
@@ -593,11 +593,14 @@ def statusFromName(name, anime=False):
     }
 
     @staticmethod
-    def should_search(status, show_obj):
+    def should_search(status, show_obj, manual_searched):
```

Comment:
@medariox because is not optional in here. in should_replace is optional because is a multi use method